odk1-src/form-repeats.rst

```diff
@@ -38,7 +38,25 @@ and set a constraint on that question.
   text, guest_name, Guest's name, ,
   text, guest_dietary, Does this guest have any dietary restrictions?, , 
   end_repeat,,,,
+
+.. note::
+
+  If the count is decreased by a user, no groups will be deleted. It's an approach we agreed on in order to avoid accidental data loss.
```

Comment:
To be a little more explicit, how about: "It's an approach we agreed on in order to avoid accidental data loss." -> "This avoids accidental data loss: a user who accidentally sets the count too low can set it to a higher number and still have the repetitions that were previously created."

Reply:
Improved.